canaries/main.dart

```diff
@@ -0,0 +1,137 @@
+import 'package:flutter/material.dart';
+import 'package:amplify_flutter/amplify_flutter.dart';
+import 'amplifyconfiguration.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  // This widget is the root of your application.
```

Comment:
Would it be more useful to copy over the code from the app in `/example`? Not sure what the goal is here. The only category dependency is datastore so it won't catch issues in any other category.